filesystem/lvm2.ksy

```diff
@@ -0,0 +1,164 @@
+meta:
+  title: Logical Volume Manager version 2
+  id: lvm2
+  endian: le
+  encoding: ascii
+  application:
+    - linux
+    - grub2
+    - lvm tools
+    - libvslvm
+  license: GFDL-1.3+
+doc: |
+  === Building a test file
+  ....
+  dd if=/dev/zero of=image.img bs=512 count=$(( 4 * 1024 * 2 ))
+  sudo losetup /dev/loop1 image.img
+  sudo pvcreate /dev/loop1
+  sudo vgcreate vg_test /dev/loop1
+  sudo lvcreate --name lv_test1 vg_test
+  sudo losetup -d /dev/loop1
+  ```
+doc-ref: https://github.com/libyal/libvslvm/blob/master/documentation/Logical%20Volume%20Manager%20(LVM)%20format.asciidoc
+instances:
+  sector_size:
+    value: 512 # TODO: how about 4k sectors?
+seq:
+  - id: pv
+    type: physical_volume
+types:
+  physical_volume:
+    seq:
+      - id: empty_sector
+        size: _root.sector_size 
+      - id: label
+        type: label
+    types:
+      label:
+        seq:
+          - id: label_header
+            type: label_header
+          - id: volume_header
+            type: volume_header
+        types:
+          label_header:
+            seq:
+              - id: signature
+                contents: "LABELONE"
+              - id: sector_number
+                type: u8
+                doc: "The sector number of the physical volume label header"
+              - id: checksum
+                type: u4
+                doc: "CRC-32 for offset 20 to end of the physical volume label sector"
+              - id: label_header_
+                type: label_header_
```

Comment:
Um, probably a bad idea to use names like that — both `label_header` and `label_header_` would be transformed into `LabelHeader` in camel case style, and will clash.